src/rvoice/fluid_rvoice_event.h

```diff
@@ -89,16 +82,15 @@ fluid_rvoice_eventhandler_get_finished_voice(fluid_rvoice_eventhandler_t* handle
 
 
 int fluid_rvoice_eventhandler_push(fluid_rvoice_eventhandler_t* handler, 
```

Comment:
Maybe rename this to ```fluid_rvoice_eventhandler_push_int_real``` and ```fluid_rvoice_eventhandler_push_param``` to ```fluid_rvoice_eventhandler_push```. The latter is now the general case and could do with a generic name, I think.